packages/core/src/exporter.ts

```
@@ -245,138 +172,147 @@ export const fillValue = async (
 
-      value = branches.map(branch => branch.title).join(", ");
+      value = branches.map((branch) => branch.title).join(', ');
 
       break;
 
-    case "departments":
+    case 'departments':
       const departments = await models.Departments.find({
-        _id: item.departmentIds
+        _id: item.departmentIds,
       }).lean();
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)